dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tag.java

```diff
@@ -198,18 +215,19 @@ public boolean equals(Object obj) {
             return false;
         }
         final Tag other = (Tag)obj;
-        return Objects.equals(this.getId(), other.getId());
+        return Objects.equals(this.getId(), other.getId()) && Objects.equals(this.lastBuilt, other.lastBuilt);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, name, reference);
+        return Objects.hash(id, name, reference, lastBuilt);
```

Comment:
See comment in `equals` -- I think you can just return the hash of the id.

Reply:
continuing pattern as denis suggested